### ISSUE TITLE
Sharing Service: Unable to get clients to connect to server

### ISSUE THREAD

**Issue by HodgsonSDAS** [Microsoft/HoloToolkit-Unity#507]
Currently unable to build the SharingTest scene get HoloLens devices to connect to the sharing server over a wireless local area network.

Repro Steps:

1. Open `SharingTest.scene`
2. Start Sharing Service.
3. Copy Sharing Service IP address from server log and paste into Sharing Stage: Server Address field on your `SharingStage` scene object.
4. Build Settings -> Add current scene. 
5. Build and deploy to HoloLens via Remote Machine debugger.
6. Output shows it never connects, and server log never registers device connects.

> Note: I tried both local and external IP and made sure all the devices were connected to the same wireless network. I'm also able to connect to the server via the editor using localhost, and specified IPs.

**Comment by NeerajW** [comment]
So it worked over WiFi?

**Comment by HodgsonSDAS** [comment]
Does not work via WiFi.

**Comment by NeerajW** [comment]
It would be worth investigating which changes broke this. Was it the big refactor stuff or something later than that. Sorry I am stuck with snow and don't have a device at the moment :(

**Comment by HodgsonSDAS** [comment]
No worries, I'm pretty sure it's the binaries that are broken.

**Comment by NeerajW** [comment]
@matthejo might be able to help with specifics.

**Comment by HodgsonSDAS** [comment]
Actually... Seems like my firewall was blocking it.  Thought that it was configured properly.

Never got a new prompt to allow the Sharing Service through the firewall.
Gonna keep on with more testing, and might add that in the documentation.

**Comment by sebrk** [comment]
@NeerajW I've tried about everything. I wrote my own UDP client/server. Works flawless in Windows 10. Not in Hololens. I've tried other ways of communicating with the Hololens but nothing seems to work. Except using your SharingService.exe. However it is way to feature heavy for what I want to do: simply change scenes using an Android device. How can I get a SharingService-like thing working on a mobile device? I cannot find the code for SharingService? Or are we simply bound to use this setup, involving a PC hosting the service?
